pkg/controller/nstemplateset/nstemplateset_controller.go

```diff
@@ -452,200 +235,38 @@ Current:
 	return deleted, nil
 }
 
+// fetchNamespaces returns all current namespaces belonging to the given user
+// i.e., labeled with `"toolchain.dev.openshift.com/owner":<username>`
+func fetchNamespaces(client client.Client, username string) ([]corev1.Namespace, error) {
```

Comment:
Do you think we can/should move it to namespaces.go too?

Reply:
whoops, that was somehow forgotten :man_shrugging:  moved in 34e547a